diagnostic.md

```diff
@@ -6,70 +6,78 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+def show
+  @book = Book.find(4)
+
+  render json: @book
+end
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+app/controllers/books_controller.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+Migrate takes an existing database and applies a change or changes to it.
```

Comment:
`bundle exec rake db:migrate` executes all of the database migration files in `db/migrate` and generates a new database schema file, `db/schema.rb`.